src/UI/components/shared/Notauthenticated.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@mui/material'
+import React from 'react'
+import { useRouter } from 'next/router';
+
+interface authrole{
+  name: string
+}
+
+function Notauthenticated ({name}:authrole)  {
+    const router = useRouter();
+    const backHome =  () => router.push('./')
+  return (
+    <div>
+        <h1>You Are Not An {name}</h1>
+        <p><a href="vectoratlas@outlook.com">visit this link to request permision</a></p>
```

Comment:
does this need to be a mail to link? Something like `mailto:vectoratlas@outlook.com`

Reply:
it would still be a link, I think it's just the href that needs updating